docs/README.md

```diff
@@ -28,7 +28,8 @@ You are now able to use all blocks that are exported by the `condition-layout` a
 
 | Block name                 | Description                                                                                                                  |
 | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| `condition-layout.product` | Defines the condition logic and the children blocks that are going to be rendered in case the predefined conditions are met. |
+| `condition-layout.product` | Defines the condition logic on the product context and the children blocks that are going to be rendered in case the predefined conditions are met. |
+| `condition-layout.binding` | Defines the condition logic on the current binding and the children blocks that are going to be rendered in case the predefined conditions are met.  |
 
 ### Step 2 - Adding the `condition-layout.product` block to your theme's templates
```

Comment:
It would be nice to also add an example of the `condition-layout.binding` here. I can do that! Could you send me the JSON?

Reply:
Agree, I will add an example in the commit

Reply:
Hi, @guerreirobeatriz 
I added code examples in the readme. Can you double check please?